.travis.yml

```diff
@@ -16,7 +17,7 @@ matrix:
     - php: 7.1
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 7.4
-      env: STABILITY=dev
+    - php: 8.0
```

Comment:
Could you please revert this change? 

It is not needed, but we should test dev-stability on PHP7.4

Reply:
done

Reply:
This line can be removed as well. You've added `8.0` in line 8 already, that should be enough.

Reply:
done, thanks for review